service/iot-service-samples/pnp-service-sample/temperature-controller-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/service/TemperatureController.java

```diff
@@ -61,7 +64,10 @@ private static void GetAndUpdateTwin() throws IOException, IotHubException {
         String propertyName = "targetTemperature";
         double propertyValue = 60.2;
         String componentName = "thermostat1";
-        twin.setDesiredProperties(PnpHelper.CreateComponentPropertyPatch(propertyName, propertyValue, componentName));
+        JsonObject patchJson = new JsonObject();
```

Comment:
Can you add the comment we had in the helper which was showing how the patch json should look like?

Reply:
coming in next commit